include/dlaf/eigensolver/bt_reduction_to_band/impl.h

```diff
@@ -317,12 +315,11 @@ void BackTransformationReductionToBand<B, D, T>::call(
         }
       }
 
+      const GlobalTileIndex taus_index{Coord::Col, k};
```

Comment:
@albestro, @aurianer, @rasolca, @RMeli, I've used the `Coord::X` constructor version in some places for `taus` indices, but I haven't been consistent with them yet (other places use e.g. `*Index{0, k}`. Do you prefer the `Coord` version generally? Eventually we still need some sort of `Vector` to abstract away this use case, but I will not add that in this PR.